docs/drivers.md

```diff
@@ -3,26 +3,98 @@ Bit components are language agnostic. You can write components in any language y
 
 ### What does the driver do?
 
-The driver resolves the ID of the components you want to use in your code, and using [dependency injection](https://en.wikipedia.org/wiki/Dependency_injection) run the component you want to use.
+Drivers are responsible for several tasks:
+
+1. Resolving the `bit.json` file, and fetching all components and dependencies to the `components` directory.
+2. Creating and maintainig all relevant links for the language-specific development environment, with the components imported with Bit.
+  * For example - in JavaScript, to use a module via `require`, the module will need to be in the `node_modules` direcotry, so in this case the driver maintains a mock-module with links back to the `components` directory, where all components are stored.
 
 **Important** To import and manage components use Bit-CLI. This doc only explains how to use components in your code.
 
 In this docs we will describe all currently available drivers.
 
+### Drivers are simply devDependencies
+
+Bit drivers are only required when developing or building projects. They are there simply to create the virtual library. Once the virtual library is set accoring to your project's bit.json file, your application will natively use the components, just like libraries.
+
+The only thing you need to do is to remember to pack the `components` directory alongside your project.
+
+### Component devDependencies
+
+Bit manages all of it's components devDependencies in it's own `components` folder, so not to interfere with your project's environment.
```

Comment:
Bit manages all of **its** **component's** devDependencies in **its** own `components` **directory**.

(maybe we should talk directly about `node_modules`/`package dependencies`, because devDependencies is not a term we use in our product)